src/components/Navbars/IndexNavbar.js

```diff
@@ -69,9 +71,9 @@ function IndexNavbar() {
           <NavbarBrand
             data-placement="bottom"
             href="/"
-            title="Coded by Creative Tim"
-          >
-            speed hiring
+            title="Coded by Creative Tim">
+            
+            <img src={logo} className="logo"></img>
```

Comment:
No console está surgindo uma sugestão de colocarmos o alt na tag img
"img elements must have an alt prop, either with meaningful text, or an empty string for decorative images"